### PR TITLE
Updated Programmatic Schemas section

### PR DESCRIPTION
**Pull request by dupski**
Added notes on how to make a programmatically-generated schema executable

### CODE REVIEW

File: content/documentation/master/schema.md [hunk 1]
```diff
@@ -258,7 +258,25 @@ Example:
...
+
+    GraphQLObjectType queryType = newObject()
+            .name("QueryType")
+            .field(newFieldDefinition()
+                    .name("hello")
+                    .type(GraphQLString)
+                    .dataFetcher(new StaticDataFetcher("world!"))
+            .build();
+
+    GraphQLSchema schema = new GraphQLSchema(queryType);
```
Comment (bbakerman) [inline content/documentation/master/schema.md]: use the builder - not the direct constructor

File: content/documentation/v11/schema.md [hunk 2]
```diff
@@ -258,7 +258,25 @@ Example:
...
+
+    GraphQLObjectType queryType = newObject()
+            .name("QueryType")
+            .field(newFieldDefinition()
+                    .name("hello")
+                    .type(GraphQLString)
+                    .dataFetcher(new StaticDataFetcher("world!"))
+            .build();
+
+    GraphQLSchema schema = new GraphQLSchema(queryType);
```
Comment (bbakerman) [inline content/documentation/v11/schema.md]: use builder

File: content/documentation/master/schema.md (hunk 1 above)
Comment (dupski) [reply content/documentation/master/schema.md]: Done. It would be nice if it was in the documentation....... :)

**Comment by andimarek** [comment]
@bbakerman are all of you concerns addressed?

**Pull request merged**